helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ZkClientPathMonitor.java

```diff
@@ -143,6 +150,10 @@ public ZkClientPathMonitor(PredefinedPath path, String monitorType, String monit
         new HistogramDynamicMetric(PredefinedMetricDomains.DataPropagationLatencyGauge.name(),
             new Histogram(new SlidingTimeWindowArrayReservoir(getResetIntervalInMs(),
                 TimeUnit.MILLISECONDS)));
+    _dataPropagationLatencyGuage =
+        new HistogramDynamicMetric(PredefinedMetricDomains.DataPropagationLatencyGuage.name(),
+        new Histogram(new SlidingTimeWindowArrayReservoir(getResetIntervalInMs(),
+            TimeUnit.MILLISECONDS)));
```

Comment:
Could we add an inline comment saying this is deprecated and keeping it for backward-compatibility purposes?